api/v1alpha1/operandconfig_types.go

```diff
@@ -39,6 +39,34 @@ type ConfigService struct {
 	Spec map[string]runtime.RawExtension `json:"spec"`
 	// State is a flag to enable or disable service.
 	State string `json:"state,omitempty"`
+	// Resources is used to specify the k8s resources that are needed for the service.
+	// +optional
+	Resources []ConfigResource `json:"resources,omitempty"`
+}
+
+// ConfigResource defines the resource needed for the service
+type ConfigResource struct {
+	// Name is the resource name.
+	Name string `json:"name"`
+	// Kind identifies the kind of the custom resource.
```

Comment:
I believe `custom resource` is not accurate. How about `kubernetes resource`?

Reply:
Great Catch, updated!